app/src/main/java/com/gholem/moneylab/features/add/adapter/TransactionsAddAdapter.kt

```diff
@@ -0,0 +1,81 @@
+package com.gholem.moneylab.features.add.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.gholem.moneylab.R
+import com.gholem.moneylab.domain.model.AddFragmentDataModel
+
+class TransactionsAddAdapter : RecyclerView.Adapter<TransactionsAddAdapter.ViewHolder>() {
```

Comment:
Change the name to AddTransactionAdapter or as I suggested before if you want to use 'Expense' instead of 'Transaction' then use 'AddExpenseAdapter'